docs/cpp/source/install.rst

```diff
@@ -78,36 +78,51 @@ Your target's files will be linked with the library and can include headers (e.g
 Installing Python Bindings
 --------------------------
 
-Installing
-~~~~~~~~~~
+The python package may be installed with ``pip``:
 
-Set up your environment:
+.. code-block:: shell
+
+  pip install gtn
+
+To build and install from source, first clone the repo:
+
+.. code-block:: shell
+
+  git clone https://github.com/facebookresearch/gtn.git
+
+Setup your environment:
+
+.. code-block:: shell
+
+  conda create -n gtn_env
+  conda activate gtn_env
+
+Install dependencies:
 
 .. code-block:: shell
 
-    conda create -n gtn_env
-    conda activate gtn_env
+  cd bindings/python
```

Comment:
Is it possible to share rst sources between Sphinx projects? Might be nice if we could have one file for both of these and just inject it here.

Reply:
Yea...I was hoping in fact to have the python docs be included in the same build as the cpp docs, but I didn't figure out how to have to configs (for the two different languages) work well. That would be ideal and then relative links would be easy within the same system.